pcsx2/gui/AppCoreThread.cpp

```diff
@@ -406,30 +422,35 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	ResetCheatsCount();
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length()) {
-		if (EmuConfig.EnableWideScreenPatches || EmuConfig.EnableCheats) {
+	if (!gameCRC.Length())
+	{
+		if (EmuConfig.EnableWideScreenPatches || EmuConfig.EnableCheats)
+		{
 			Console.WriteLn(Color_Gray, "Patches: No CRC, using 00000000 instead.");
 		}
 		gameCRC = L"00000000";
 	}
 
 	// regular cheat patches
-	if (EmuConfig.EnableCheats) {
-		if (numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats")) {
+	if (EmuConfig.EnableCheats && (numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats")))
+	{
```

Comment:
my bad, it seems you were [right](https://github.com/PCSX2/pcsx2/pull/1266#discussion_r57924312). Initially I had thought declaration would produce the same warnings mentioned [over here](https://github.com/PCSX2/pcsx2/pull/1266#discussion_r57914220). I'll change the code later. it's past my bed time right now.